### ISSUE TITLE
[Support]: Docker run command is not correctly generated

### ISSUE THREAD

**Issue by Daniel-Klockenkaemper** [github/gh-actions-importer#6]
### What operating system are you using?

- [X] Linux
- [ ] macOS
- [ ] Windows

### What version of the tool are you using?

Latest Beta

### What happened?

The Actions Importer GH CLI Plugin does not correctly generates the docker run command. Every command errors with a "docker: invalid reference format" error message.

When manually generating the docker run command, it works as expected:

```shell
docker run -e "GHCR_USERNAME=XXX" -e "GHCR_PASSWORD=XXX" -e "GITHUB_ACCESS_TOKEN=XXX" -e "GITHUB_INSTANCE_URL=https://github.com" -e "GITLAB_ACCESS_TOKEN=XXX" -e "GITLAB_INSTANCE_URL=https://gitlab.XXX" -e "http_proxy=$http_proxy" -e "HTTP_PROXY=$http_proxy" -e "HTTPS_PROXY=$http_proxy" -v $(pwd):/data ghcr.io/actions-importer/cli:latest dry-run gitlab --namespace example-namespace --project example-project --output-dir tmp --no-ssl-verify
[2022-11-16 13:33:10] Logs: 'tmp/log/valet-20221116-133310.log'
[2022-11-16 13:33:12] Output file(s):
[2022-11-16 13:33:12]   tmp/example-namespace/example-project/.github/workflows/example-project.yml
```

### Relevant log output

```shell
gh actions-importer dry-run gitlab --namespace example-namespace --project example-project --output-dir tmp --no-ssl-verify
docker: invalid reference format.
See 'docker run --help'.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ethanis** [comment]
@Daniel-Klockenkaemper sorry you're running into this! Unfortunately, I'm not able to reproduce this issue in `gh actions-importer` and will need a bit more information. Can you provide the folliowing?

- [ ] The output of `gh actions-importer version`
- [ ] The output of `docker info`
- [ ] The output of `echo $(pwd)`

**Comment by Daniel-Klockenkaemper** [comment]
Hey, I've updated today to the latest version of gh-actions-importer, but this makes no difference. 

So here are the outputs (IP addresses of proxy configurations are masked):

```
~ gh actions-importer version
gh version 2.20.2 (2022-11-15)
gh actions-importer     github/gh-actions-importer      v1.0
actions-importer/cli    v1.0.14291

 ~ docker info                                       
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Docker Buildx (Docker Inc., v0.8.2-docker)
  scan: Docker Scan (Docker Inc., v0.17.0)

Server:
 Containers: 18
  Running: 0
  Paused: 0
  Stopped: 18
 Images: 2
 Server Version: 20.10.16
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 212e8b6fa2f44b9c21b2798135fc6fb7c53efc16
 runc version: v1.1.1-0-g52de29d
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.15.0-52-generic
 Operating System: Ubuntu 22.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 6
 Total Memory: 15.63GiB
 Name: shmox
 ID: JBBQ:WJFY:5A2D:3ASP:TRB2:GFKA:HIV5:D63H:OE6J:7J4E:CKWF:KS5O
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http://XXX:XXXX
 HTTPS Proxy: http://XXX:XXXX
 No Proxy: XXXX
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

~ echo $(pwd)                                                                                                               
/home/dk/workdir/legacy-gitlab/strikeprice

```

**Comment by j-dunham** [comment]
Hi @Daniel-Klockenkaemper  could you please provide the contents of the file `.env.local`?  Make sure to mask the sensitive values. As a experiment could you temporarily rename the `.env.local` file to something different like `.env.local.test`, rerun the failing command, and rename the file back to `.env.local` once the command is finished?  I am not able to recreate this issue on my end yet and just want to try and eliminate that file as a cause.  With the file missing the command will not work but the docker container should at least run.

**Comment by Daniel-Klockenkaemper** [comment]
Hey, the `.env.local` file has the following values:

```
GHCR_PASSWORD=XXX
GHCR_USERNAME=daniel.klockenkaemper@XXX.com
GITLAB_ACCESS_TOKEN=XXX
GITHUB_INSTANCE_URL=https://github.com
GITHUB_ACCESS_TOKEN=XXX
GITLAB_INSTANCE_URL=https://gitlab.XXX.com
```

**Comment by Daniel-Klockenkaemper** [comment]
Ah, i have figured out the problem:

The no_proxy/NO_PROXY environment variable was somehow malformed: it had **spaces** between the single values:

This **did not** work:
```
NO_PROXY=localhost, 127.0.0.1, ::1, .XXX.de, .YYY.de
```

This **did** work:
```
NO_PROXY=localhost,127.0.0.1,::1,.XXX.de,.YYY.de
```

So, on my side it is solved! But maybe you should put the values you are sourcing from the environment into quotes, when executing the docker run command?

Thank you for your support!

**Comment by ethanis** [comment]
@Daniel-Klockenkaemper good find! This will be addressed with #11 😄

**Comment by ethanis** [comment]
This change has shipped. Can you update the CLI extension (`gh extension upgrade actions-importer`) and verify it's working as expected?
